lib/matplotlib/tests/test_image.py

```diff
@@ -230,14 +245,13 @@ def test_cursor_data():
 
 @image_comparison(baseline_images=['image_clip'])
 def test_image_clip():
-    from math import pi
-
-    fig = plt.figure()
-    ax = fig.add_subplot(111, projection='hammer')
+    d = [[1, 2], [3, 4]]
 
-    d = [[1,2],[3,4]]
+    fig, ax = plt.subplots()
+    im = ax.imshow(d)
+    patch = patches.Circle((0, 0), radius=1, transform=ax.transData)
+    im.set_clip_path(patch)
 
-    im = ax.imshow(d, extent=(-pi,pi,-pi/2,pi/2))
```

Comment:
Why the change? Seemed like a perfectly fine test, I think. Also, the old test utilized the hammer projection while this one doesn't?